code/control/DeployPlanDispatcher.php

```diff
@@ -290,6 +284,12 @@ protected function getGitPrevDeploys($project) {
 	 * @return SS_HTTPResponse
 	 */
 	protected function getAPIResponse($output, $statusCode) {
+		$secToken = $this->getSecurityToken();
+		$secToken->reset();
```

Comment:
I thought about this.  

1) I remove the regeneration of the CSRF on responses to GET requests
2) we can only make one POST request at the same time to the same dispatcher.

Now I'm not really sure if I want to solve the 2) in this PR, or if it's going to be a common problem. Hopefully we can break up potential POST actions into their own dispatchers.